pkgs/by-name/ne/nerdfetch/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitea
+}:
+stdenv.mkDerivation rec {
+  pname = "nerdfetch";
+  version = "8.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "thatonecalculator";
+    repo = "NerdFetch";
+    rev = "v${version}";
+    hash = "sha256-yV1IYUixBkiFEB2NfKdvK/8XT0BrQ5XsBME7XWNDXZU=";
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+      cp $src/nerdfetch $out/bin
+  '';
+
```

Maybe add an update script

```suggestion

  passthru.updateScript = gitUpdater {
    rev-prefix = "v";
  };
  
```